app/assets/stylesheets/section.scss

```diff
@@ -0,0 +1,7 @@
+.section {
```

Comment:
This is too broad a change as it will change every `.section` within the whole platform. I'd rather insert this into the `team-page.scss` file so the changes only affect that single page.